CHANGELOG.md

```diff
@@ -2,9 +2,45 @@
 
 ## Table of Contents
 
+- [v0.1.0][#v010]
```

Comment:
The current form won't render correctly.
```suggestion
- [v0.1.0](#v010)
```

Reply:
Updated, thanks!